xtask/src/util.rs

```diff
@@ -44,6 +44,19 @@ pub fn run_cmd(mut cmd: Command) -> Result<()> {
     }
 }
 
+/// Print a `Command` and run it, then check that it completes
+/// successfully. Return the command's stdout.
+pub fn get_cmd_stdout(mut cmd: Command) -> Result<Vec<u8>> {
```

Comment:
nit: `pub fn run_command(mut cmd: Command) -> Result<String>`

1) From `cmd.output()` it is not obvious that the command is executed here. I had to double-check that in the [docs](https://doc.rust-lang.org/std/process/struct.Command.html#method.output)
2) I think we can already check here whether we have a string, right? Anything non-utf8 should be a failure anyway, right?

Reply:
Renamed to `run_cmd_get_stdout` -- we already have a `run_cmd` above, which is useful when you want the stdout/stderr to not be hidden (like when running most cargo commands).

Changed to convert to a `String` as suggested.